.travis.yml

```diff
@@ -50,10 +50,29 @@ jobs:
       python: 3.7
 
     - stage: "integration"
+      name: "Netbox v2.10"
       before_script:
         - "pip install invoke toml poetry ansible pynetbox"
-        - "ansible-galaxy collection install netbox.netbox"
+        - "ansible-galaxy collection install git+https://github.com/netbox-community/ansible_modules.git,v1.2.0"
         - "poetry install"
+      env: "NETBOX_VERSION=v2.10"
+
+    - stage: "integration"
+      name: "Netbox v2.9"
+      before_script:
+        - "pip install invoke toml poetry ansible pynetbox"
+        - "ansible-galaxy collection install git+https://github.com/netbox-community/ansible_modules.git,v1.2.0"
+        - "poetry install"
+      env: "NETBOX_VERSION=v2.9"
```

Comment:
Do we need a `script` here as well?

Reply:
Good catch, I've made the update and after multiple tries the integration tests are now passing for 2.8, 2.9 and 2.10